docs/source/guides/snippets/grpc/.bazelignore

```diff
@@ -0,0 +1,5 @@
+java/src/main/java/com/bentoml
```

Comment:
move `snipperts/grpc` to a top-level folder `grpc-client`?

Reply:
should we move it to examples/grpc-client?